tests/resources/Vsphere-UI-Util.robot

```diff
@@ -50,3 +50,108 @@ Navigate To VCH Creation Wizard
     Wait Until Page Contains  Summary
     Wait Until Page Contains  Virtual Container Hosts
     Wait Until Page Contains  Containers
+
+Prepare Testbed For Protractor Tests
+    # Force the WD to be vic-ui/
+    ${wd}=  Run  pwd
+    @{wd_split}=  Split String  ${wd}  /
+    ${wd_level_len}=  Get Length  ${wd_split}
+    ${last_level_idx}=  Evaluate  ${wd_level_len} - 1
+    ${current_folder}=  Get From List  ${wd_split}  ${last_level_idx}
+    Run Keyword Unless  '${current_folder}' == 'vic-ui'  Fatal Error  Test script should be run from vic-ui/
+
+    # Check drone CLI
+    ${rc}  ${drone_ver}=  Run And Return Rc And Output  drone --version 2>&1
+    Log  Checking Drone version...
+    Log  return code: ${rc}, output: ${drone_ver}  DEBUG
+    Run Keyword If  ${rc} > ${0}  Fatal Error  Drone is required to run tests!
+    Run Keyword If  '0.5.0' not in '${drone_ver}'  Fatal Error  Drone 0.5.0 is required to run tests!
+
+    # Check govc
+    ${rc}=  Run And Return Rc  govc
+    Should Be True  ${rc} != 127
```

Comment:
What's 127?

Reply:
it's the RC you get when you try to run a binary that doesn't exist

Reply:
Ok. This seems very much a 'don't ever update this' value so either a note or put into a constant would be appropriate.

Reply:
i don't see much value in setting a constant in framework like this but leaving a comment sounds reasonable here